contracts/airdrops/Existing1155NftDrop.sol

```diff
@@ -87,20 +85,14 @@ contract Existing1155NftDrop is AirdropInfo, AirdropMerkleProof, IERC1155Receive
         airdropFunded = true;
         airdropFundBlockTimestamp = block.timestamp;
         airdropFundingHolder = msg.sender;
-        emit AirdropFunded();
+        emit AirdropFunded(address(this));
```

Comment:
I don't understand why emit event with address(this)

Reply:
Someone on the team that is working on the frontend side said that they need the address of the contract along with that event (while catching that event) so they can pass that address somewhere else. Do you know of a better way to do this? @bxmmm1

Reply:
In order to listen / query events on a smart contract, you need the smart contract address. 
Maybe I'm mistaken, but what I see here is:
``` 
I'm listening for events on contract 0x123 and when the event AirdropFunded happens
it will give me what I need which is 0x123
```

Reply:
I get what you're saying, it was weird to me too hahaha. I'll check with the team about that today.